pages/linux/lxi.md

```diff
@@ -0,0 +1,24 @@
+# lxi
+
+> Control LXI compatible instruments such as oscilloscopes.
+> More information: <https://github.com/lxi-tools/lxi-tools>.
+
+- Discover LXI devices on available networks:
+
+`lxi discover`
+
+- Capture a screenshot using plugin autodetection:
```

Comment:
Is the plugin called `autodetection` or does it autodetect plugins?

Reply:
In case it's called `autodetection`, I'd say
```suggestion
- Capture a screenshot using the autodetection plugin:
```

Reply:
It does plugin autodetection

Reply:
Wait for @marchersimon to get a reply before committing